core/src/validator.rs

```diff
@@ -1366,6 +1367,15 @@ fn post_process_restored_tower(
     Ok(restored_tower)
 }
 
+fn blockstore_options_from_config(config: &ValidatorConfig) -> BlockstoreOptions {
```

Comment:
I made this common function to avoid duplicate code. It is currently in a separate commit, so happy to break this out if folks want.